### PR TITLE
Add static_dns workspace

### PR DESCRIPTION
**Pull request by ludeeus**
Add static_dns workspace and initialize with a `_checkdns` record

### CODE REVIEW

File: static_dns/dns.tf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+resource "cloudflare_record" "_checkdns" {
+  zone_id = data.cloudflare_zone.dns_zone.id
+  name    = "_checkdns"
+  value   = "127.0.0.1"
```
Comment (ludeeus) [inline static_dns/dns.tf]: It needs to have a value, as this is a dummy record `127.0.0.1` seems sensible?

File: static_dns/dns.tf [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
```
Comment (ludeeus) [inline static_dns/dns.tf]: Should this file be named `records.tf` maybe?

Comment (pvizeli) [reply static_dns/dns.tf]: make sense

File: static_dns/dns.tf (hunk 1 above)
Comment (pvizeli) [reply static_dns/dns.tf]: I guess that works fine, we can also use `1.1.1.1` if you like that more or `10.0.0.0`

**Review by pvizeli — APPROVED** [review]

**Pull request merged**